translate.js

```diff
@@ -8,7 +8,7 @@
 /// New style: 0.4.5+commit.b318366e.Emscripten.clang
 function versionToSemver (version) {
   // FIXME: parse more detail, but this is a good start
-  var parsed = version.match(/^([0-9]+\.[0-9]+\.[0-9]+)-([0-9a-f]{8})[\/\*].*$/);
```

Comment:
I don't understand the description of the commit. This is just about not escaping the special symbols, which is not required here anyway.

Reply:
That's it.